core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -266,6 +266,7 @@ object SparkSubmit {
       }
     }
 
+
```

Comment:
nit: I'm going to nix this blank line when I merge (no action required on your part)